espnet2/svs/xiaoice/XiaoiceSing.py

```diff
@@ -455,7 +484,9 @@ def forward(
         midi = midi[:, : midi_lengths.max()]  # for data-parallel
         label = label[:, : label_lengths.max()]  # for data-parallel
         tempo = tempo[:, : tempo_lengths.max()]  # for data-parallel
-
+        if self.loss_function == "XiaoiceSing2":
+            log_f0 = pitch[:, : pitch_lengths.max()]
+            vuv = log_f0 != 0
```

Comment:
For log f0, why you are checking the 0 value? Isn't should be to check whether it is negative or positive?

Reply:
Fixed it by setting negative log_f0 to zero.